opm/output/eclipse/Summary.cpp

```diff
@@ -905,7 +907,10 @@ Summary::Summary( const EclipseState& st,
 
 	    this->handlers->misc_nodes.emplace( keyword, nodeptr ); 
         } else {
-	    if( funs.find( keyword ) == funs.end() ) continue;
+	        if( funs.find( keyword ) == funs.end() ) {
```

Comment:
I guess the call to `OpmLog`could come directly here?

Reply:
The keyword list includes well-specific items. That is, include "WOPR" and with 3 wells (and WOPR unsupported) you would get 3 warnings (stating that WOPR is unused..).

Reply:
Good point